stl/inc/ranges

```diff
@@ -512,10 +512,10 @@ namespace ranges {
         }
 
         // clang-format off
-        _Defaultabox(_Defaultabox&&) requires is_trivially_move_constructible_v<_Ty> = default;
+        _Defaultabox(_Defaultabox&&)  noexcept requires is_trivially_move_constructible_v<_Ty> = default;
         // clang-format on
 
-        constexpr _Defaultabox(_Defaultabox&& _That) : _Engaged{_That._Engaged} {
+        constexpr _Defaultabox(_Defaultabox&& _That) noexcept : _Engaged{_That._Engaged} {

```

Comment:
Missing exception specification here is probably a bug. `_Defaultabox` is in for `join_view::_Iterator` and `lazy_split_view`, while the standard wording uses `optional` and _`non-propagating-cache`_ respectively. However, I'm not sure whether it's OK to use `_Defaultabox` in these 2 cases.